src/components/number_input.tsx

```diff
@@ -24,13 +26,28 @@ interface NumberInputProps {
  * input value. The function has the new input value as the sole arguement. 
  * @param value The input value
  */
-const NumberInput: FC<NumberInputProps> = ({id, label, prefix="", suffix="", updateValue, value}) => {
+const NumberInput: FC<NumberInputProps> = ({id, label, prefix="", suffix="", updateValue, value, min, max}) => {
+  const [error, setError] = useState<string>('');
+
+  const handleChange = (inputValue : string) => {
+    if (typeof inputValue !== 'number') {
```

Comment:
Is this possible for a `type='number'` input?

Reply:
I think so, this part is sort of wrong, I'll fix it in the next push. But I do think it is possible we get a invalid input

Reply:
Yes, I see that the type of inputValue is string, so line 33 would always fail.  I assume to mean to make sure the string converts to a proper number.

Reply:
checkout the latest commit on this, I think I already fixed that